elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/compile/ElideDynamicEntityCompiler.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.compile;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurityConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideTableConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.parser.ElideConfigParser;
+import com.yahoo.elide.contrib.dynamicconfighelpers.parser.handlebars.HandlebarsHydrator;
+
+import com.google.common.collect.Sets;
+
+import org.mdkt.compiler.InMemoryJavaCompiler;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+/**
+ * Compiles dynamic model pojos generated from hjson files.
+ *
+ */
+@Slf4j
+public class ElideDynamicEntityCompiler {
+
+    public static ArrayList<String> classNames = new ArrayList<String>();
+
+    public static final String PACKAGE_NAME = "com.yahoo.elide.contrib.dynamicconfig.model.";
```

Comment:
Prefixing the models with "com.yahoo.elide" will prohibit API versioning in Elide 5 (all elide prefixed models are not versioned).  We'll need to find another package name for the dynamic models.

Reply:
we can use dynamicconfig.models